rqpy/plotting/_iv_didv_tools_plotting.py

```diff
@@ -257,3 +323,455 @@ def _plot_energy_res_vs_bias(r0s,
+            np.sqrt(np.abs(data.noise_model['isquid'][2][ii])),
+            alpha=.5,
+            color='#9467bd',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(psd)),
+            color='#8c564b',
+            alpha=0.8,
+            label='Raw Data',
+        )
+        ax.set_ylabel(
+            r'Input Referenced Current Noise [A/$\sqrt{\mathrm{Hz}}$]'
+        )
+
+        lgd = plt.legend(loc='upper right')
+
+        fig, ax = plt.subplots(1,1, figsize=(11,6))
+        if ylims_power is not None:
+            ax.set_ylim(ylims_power)
+        if xlims is not None:
+            ax.set_xlim(xlims)
+
+
+        ax.grid(which="major", linestyle='--')
+        ax.grid(which="minor", linestyle="dotted", alpha=0.5)
+        ax.tick_params(which="both", direction="in", right=True, top=True)
+        ax.set_xlabel(r'Frequency [Hz]')
+
+        ax.set_title(f"Power Noise For $R_0$ : {r0*1e3:.2f} $m\Omega$")
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ptes'][0][ii])),
+            color='#1f77b4',
+            linewidth=1.5,
+            label='TES Johnson Noise'.
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ptes'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['ptes'][2][ii])),
+            alpha=.5,
+            color='#1f77b4',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['pload'][0][ii])),
+            color='#ff7f0e',
+            linewidth=1.5,
+            label='Load Noise',
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['pload'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['pload'][2][ii])),
+            alpha=.5,
+            color='#ff7f0e',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ptfn'][0][ii])),
+            color='#2ca02c',
+            linewidth=1.5,
+            label='TFN Noise',
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ptfn'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['ptfn'][2][ii])),
+            alpha=.5,
+            color='#2ca02c',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ptot'][0][ii])),
+            color='#d62728',
+            linewidth=1.5,
+            label='Total Noise',
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ptot'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['ptot'][2][ii])),
+            alpha=.5,
+            color='#d62728',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['psquid'][0][ii])),
+            color='#9467bd',
+            linewidth=1.5,
+            label='Squid+Electronics Noise',
```

Comment:
Perhaps we should replace the `+` with an ampersand `&`? I feel that the plus sign is a little colloquial for plots.
```suggestion
            label='Squid & Electronics Noise',
```

Reply:
added